ChangeLog

```diff
@@ -23,6 +23,7 @@ Bugfix
    * Fix propagation of restart contexts in restartable EC operations.
      This could previously lead to segmentation faults in builds using an
      address-sanitizer and enabling but not using MBEDTLS_ECP_RESTARTABLE.
+   * Fix `mbedtls_platform_zeroize()` to check for zero length and NULL pointer.
```

Comment:
The changelog entry must mention the issue (https://github.com/ARMmbed/mbed-crypto/issues/49). It should also explain _why_ the change was made, rather than _what_ was changed. And it isn't markdown. Suggestion:

> Fix possible undefined behavior in mbedtls_platform_zeroize() with a 0-length buffer. #49